services/dynamic-sidecar/Dockerfile

```diff
@@ -99,24 +104,26 @@ RUN pip --no-cache-dir install -r requirements/prod.txt &&\
 FROM base as production
 
 ENV SC_BUILD_TARGET=production \
-    SC_BOOT_MODE=production
+  SC_BOOT_MODE=production
 
 ENV PYTHONOPTIMIZE=TRUE
 
 WORKDIR /home/scu
 
 # Starting from clean base image, copies pre-installed virtualenv from prod-only-deps
 COPY --chown=scu:scu --from=prod-only-deps  ${VIRTUAL_ENV} ${VIRTUAL_ENV}
+COPY --chown=scu:scu --from=prod-only-deps  ${DY_VOLUMES} ${DY_VOLUMES}
 
 # Copies booting scripts
 COPY --chown=scu:scu services/dynamic-sidecar/docker services/dynamic-sidecar/docker
 RUN chmod +x services/dynamic-sidecar/docker/*.sh
 
-HEALTHCHECK --interval=30s \
-    --timeout=20s \
-    --start-period=30s \
-    --retries=3 \
-    CMD ["python3", "services/dynamic-sidecar/docker/healthcheck.py", "http://localhost:8000/health"]
+# more agressive checking helps with detecting the searvice is ready
+HEALTHCHECK --interval=1s \
```

Comment:
I guess we'll see how the docker engines are coping with such an interval

Reply:
uhm... did not thought it could be an issue.